requirements.txt

```diff
@@ -0,0 +1,2 @@
+pre-commit==2.13.0
+pytest==6.2.4
```

Comment:
This is just a comment, but note that `pre-commit` and `pytest` are only useful for development. It is fine to have these in the `requirements.txt` here since these are only exercises and you do not have any other application dependencies. However, note that for applications it is good practice to separate development and application dependencies. There are tools that can handle this, like [Poetry](https://python-poetry.org/), but a simple approach is to have multiple `requirements.txt` for different environments. For example, `pre-commit` and `pytest` could be in `requirements_dev.txt` while application dependencies (e.g. NumPy) would be in `requirements.txt`.

Reply:
I'll keep that in mind going forward.